examples/gstreamer-player/egl_integration.rs

```diff
@@ -20,15 +20,14 @@ pub fn init<App: slint::ComponentHandle + 'static>(
     app.window().set_rendering_notifier({
         let pipeline = pipeline.clone();
         let app_weak = app.as_weak();
-        let bus_sender = bus_sender.clone();
 
         move |state, graphics_api| match state {
             slint::RenderingState::RenderingSetup => {
                 let app_weak = app_weak.clone();
                 slint_sink.connect(
                     graphics_api,
                     &pipeline.bus().unwrap(),
-                    &bus_sender,
+                    bus_sender.clone(),
```

Comment:
Ah I see, I didn't put in the clone here. But I compensated by adding superfluous clones.